Project.xml

```diff
@@ -74,6 +74,7 @@
 	<haxelib name="newgrounds"/>
 	<haxelib name="faxe" if='switch'/>
 	<haxelib name="polymod"/>
+	<haxelib name="discord_rpc" if='desktop'/>
```

Comment:
This will only run on a desktop build of FNF

Reply:
might want to edit the readme too

Reply:
done that just now, thanks for reminding me.